ctapipe/core/tests/test_qualityquery.py

```diff
@@ -8,33 +8,33 @@
 from ctapipe.core.traits import List
 
 
-def test_selector():
+def test_selector(subarray_prod5_paranal):
     """test the functionality of an example Selector subclass"""
 
     class ExampleQualityQuery(QualityQuery):
         """Available variables: x"""
 
         quality_criteria = List(
-            default_value=[
```

Comment:
I'd leave this at it is (it should still be supported since we treat single values as `type *` and just add a new test with the full telescope-parameters version.